libs/components/lists/src/lib/modules/filter/filter-button.component.ts

```diff
@@ -50,6 +54,7 @@ export class SkyFilterButtonComponent {
    * filter button's `aria-label` attribute to provide a text equivalent for screen readers
```

Comment:
This property does is not currently displayed in the docs at https://developer.blackbaud.com/skyux/components/filter?docs-active-tab=development. Any idea why it's not showing up?

Reply:
It was just added this week in a different PR and will show up once the docs site is updated to the new version.